### ISSUE TITLE
🛑 Zanshin API user check is down

### ISSUE THREAD

**Issue by tenchisecurity** [tenchi-security/zanshin_status#14]
In [`801ef36`](https://github.com/tenchi-security/zanshin_status/commit/801ef366a5b057dad8ccc323da2024e7e41efc98
), Zanshin API user check (https://api.zanshin.tenchisecurity.com/me) was **down**:
- HTTP code: 500
- Response time: 202 ms

**Comment by edpin** [comment]
AWS region Ohio was having problems with Lambdas. The issue is now resolved.

**Comment by asieira** [comment]
AWS timeline for API Gateway in us-east-2:

[RESOLVED] Increased API Error Rates
10:59 AM PST We are investigating increased API errors for API Gateway in the US-EAST-2 Region.
11:24 AM PST We have confirmed that the elevated API error rate is restricted to control plane operations for API Gateway Version 2 API (WebSocket and HTTP APIs).
12:20 PM PST Between 10:27 AM and 12:06 PM PST, we experienced elevated API error rate in the US-EAST-2 Region, restricted to control plane operations for API Gateway Version 2 API (WebSocket and HTTP APIs). The issue has been resolved, and the service is operating normally.

**Comment by asieira** [comment]
AWS timeline for Lambda in us-east-2:

[RESOLVED] Increased Error Rates

10:46 AM PST We are investigating increased invoke error rates in the US-EAST-2 Region.

11:10 AM PST We continue to investigate increased function invoke error rates within the US-EAST-2 Region. We are working to determine the root cause of the issue and will continue to provide updates as we make progress in resolving the issue. Other AWS services, including AWS Management Console, API Gateway, and Batch, are experiencing elevated error rates as a result of this issue.

11:34 AM PST We have identified the root cause within Lambda that is causing the increased error rates for function invocations within the US-EAST-2 Region. The subsystem responsible for executing functions is currently impaired and we are working to resolve it. We have seen some improvement in Lambda function invocation error rates and believe that this will continue as we take steps to resolve the issue. Other AWS services, including AWS Management Console, API Gateway, and Batch, are experiencing elevated error rates as a result of this issue.

12:12 PM PST Between 10:27 AM and 12:03 PM PST we experienced increased Lambda invoke error rates in the US-EAST-2 Region. It may take some additional time to process backlogged async invoke traffic that accumulated during this period. The issue has been resolved and the service is operating normally.
